runtime/j9vm/j9vmnatives.xml

```diff
@@ -193,7 +193,10 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-ex
 		<export name="_JVM_HoldsLock@12"/>
 		<export name="_JVM_InitProperties@8"/>
 		<export name="_JVM_ArrayCopy@28"/>
-		<export name="_JVM_DoPrivileged@20"/>
+		<export name="_JVM_DoPrivileged@20">
+			<exclude-if condition="spec.java9"/>
+			<include-if condition="spec.win_.*"/>
```

Comment:
@keithc-ca I'm testing out these changes and I'm finding this is causing a failure.

If I remove either 197 or 198 this error goes away. Is there an issue with combining conditions like this?

http://vmfarm.rtp.raleigh.ibm.com/build_info.php?build_id=85791